web/main/browser.ts

```diff
@@ -30,6 +43,10 @@ if (!likelyWithKeyboard(window)) {
   root.classList.add('is-hotkey-disabled')
 }
 
+window.onload = () => {
+  actualizeThemeColor(themeColorTag)
+}
+
```

Comment:
`question` Is it okay to add `window.onload` callback? Otherwise we get the `rgba(0,0,0,0)` at page loading, it will be updated to proper color only on e.g. changing tab

Reply:
Yes, just use `addEventListener('load', cb)`, since `window.onload` can be used by some other program or library.